db/db_impl/db_impl.h

```diff
@@ -1934,6 +1939,7 @@ class DBImpl : public DB {
   InstrumentedCondVar atomic_flush_install_cv_;
 
   bool wal_in_db_path_;
+  std::atomic<uint64_t> max_total_wal_size_;
```

Comment:
Put it together with `max_total_in_memory_state_`.

Reply:
ok